README.md

```diff
@@ -77,22 +94,114 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
 The CSV files both contain much of the same information.
 The file with "geostreams" in its name can be uploaded to TERRAREF's Geostreams database.  
 
+### Canopy Cover: OpenDroneMap and plot boundaries <a name="opendm_can_shp" />
+
+The following steps are used when wanting to use OpenDroneMap (ODM) to create the Orthomosaic image that's then used to create the canopy cover values.
+As with the [previous example](#om_can_shp) we will be listing the steps and then providing an example.
+
+_NOTE_: the SciF Docker image uses Docker sibling containers to run the OpenDroneMap application.
+Please read our section on [Docker Sibling Containers](#docker_sibling_containers) below to be informed of potential risks with this approach.
+
+1. Create two named Docker volumes to use for processing data; one for input files and one for output files - the same volume can be used for both if desired
+2. Copy the source drone images into a folder
+3. If using a [shapefile](#shapefile) or [GeoJSON](#geojson) file, copy those into the same folder as the drone images
+4. Copy the experiment metadata file into the same folder as the drone images
+5. Copy the folder contents of the drone images folder that was just prepared onto the input named volume
+6. Create another folder for the output folders and files
+7. Run the docker container's `odm_workflow` app specifying  either the name of the shapefile or geojson file, or the URL of they [BETYdb](#betydb) instance to query for plot boundaries, and the two named volumes
+8. Copy the resulting files off the output named volume to the local folder
+9. Clean up the named volumes
+
+#### For example: <a name="opendm_can_shp_example" />
+
+You can download a sample dataset of files (archived) with names corresponding to those listed here from CyVerse using the following command.
+Be sure to replace **<username>** and **<password>** with your CyVerse username and password.
+```bash
+curl -X GET -u '<username>:<password>' https://data.cyverse.org/dav/iplant/projects/aes/cct/diag/sample-data/scif_odm_test_data.tar.gz > scif_odm_test_data.tar.gz
+gunzip scif_odm_test_data.tar.gz
+tar -xf scif_odm_test_data.tar
+```
+
+In this example we're going to assume that we're using a shapefile named `plot_shapes.shp`, that we have our drone images in a folder named `${PWD}/IMG`, and additional data in the `experiment.yaml` file.
+
+Step 1 requires creating two named Docker volumes to use when processing.
+If you already have one or more empty named volumes you can skip this step.
+You will need to have Docker running at this point.
+```bash
+docker volume create my_input
+docker volume create my_output
+``` 
+
+Step 2 involves copying the drone images into a folder:
```

Comment:
```suggestion
```

Reply:
This and the following suggested changes will break the workflow.

Reply:
fixed in https://github.com/AgPipeline/drone-makeflow/pull/4/files#r470292286